### PR TITLE
Update FPGA examples

### PR DESCRIPTION
**Pull request by eliaskoromilas**
**What this PR does / why we need it**:

* Update the FPGA XGBoost experiment:
  * use [InAccel FPGA Operator](https://artifacthub.io/packages/helm/inaccel/fpga-operator) (>=2.5) features

* Update AWS F1 instructions:
  * from **InAccel EKS-optimized FPGA AMI** to **InAccel FPGA Operator**

### CODE REVIEW

**Review by andreyvelich — COMMENTED** [review]
Thank you for updating the FPGA example @eliaskoromilas!
/ok-to-test

What do you think about adding OWNERS file under `examples/v1beta1/fpga` ?
Then, users will know whom to ask their questions.

/assign @gaocegege @johnugeorge

**Comment by eliaskoromilas** [comment]
We plan to add more FPGA experiments.

**Review by gaocegege — COMMENTED** [review]
LGTM

Thanks for your contribution! :tada: :+1:

**Comment by johnugeorge** [comment]
I think, one cannot be a owner without being a Kubeflow member. We might need to skip owner file change for now

/lgtm

**Comment by gaocegege** [comment]
> I think, one cannot be a owner without being a Kubeflow member. We might need to skip owner file change for now

I think we can just keep it to let users know who they can ask for help when they have problems with FPGA example.

**Comment by andreyvelich** [comment]
We can add them to the Kubeflow org, what do you think @gaocegege @johnugeorge ?

**Comment by gaocegege** [comment]
I have no objection to it.

**Comment by eliaskoromilas** [comment]
rebase

File: examples/v1beta1/fpga/OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+reviewers:
+  - eliaskoromilas # inaccel
+  - jstamel        # inaccel
```
Comment (andreyvelich) [inline examples/v1beta1/fpga/OWNERS:3]: Let's create approvers list, so you can approve the PRs under this folder.

```suggestion
approvers:
  - eliaskoromilas
  - jstamel
```

Comment (eliaskoromilas) [reply examples/v1beta1/fpga/OWNERS]: sure

**Comment by andreyvelich** [comment]
Thank you for driving this!
/lgtm
/approve

**Pull request merged**